functions.php

```diff
@@ -1,8 +1,15 @@
 <?php
+/** 
+* Считает количество задач в проекте 
+* @param array $tasks Массив с задачами
+* @param string $project Название проекта
+* 
+* @return int $task_count Число задач в проекте
+*/
+
 function countTasks($tasks, $project) {
-    $tc = 0;
+    $task_count = 0;
     foreach ($tasks as $task) {
-    if ($task ['category'] === $project) {$tc++;}
-    }return $tc;
-}
-
+    if ($task ['category'] === $project) {$task_count++;}
+    }return $task_count;
```

Comment:
Напоминаю про форматирование кода 

```suggestion
        if ($task ['category'] === $project) {
            $task_count++;
        }
    }

    return $task_count;
```